languages/csharp/exercises/concept/nullability/.meta/Example.cs

```diff
@@ -2,23 +2,25 @@
 
 public static class Badge
 {
-    public static int WidthInPixels(string? text,
-                                    int? fontSizePx)
+    public static string PrintLabel(string label, int? maximumWidth)
     {
-
-        return (text??"").Length*(fontSizePx??0);
+        if (maximumWidth == null)
+        {
+            return label;
+        }
+        else
+        {
+            var output = "";
+            for(int i=0;i<label.Length;i+=maximumWidth.Value)
```

Comment:
While the `for` loop does add another prequisite (the student has to know about `for` loops), I think this is fine for this case.

Reply:
Just updated the config.json file... not sure where the list of concept names will come from...